.circleci/config.yml

```diff
@@ -18,7 +18,9 @@ jobs:
             "$CIRCLE_BUILD_URL" | tee version.json
       - store_artifacts:
           path: version.json
-
+      - run:
+          name: Move example data to data dir
+          command: if [ ! -d "data" ]; then mkdir data; cp -a -v example-data/. data; fi
```

Comment:
Since the data folder is ignored to build the docker image, I believe you could skip this. Also `cp -R example-data data` should be enough (will create the dir etc.)